rules/sinks/leakages/logs/go.yaml

```diff
@@ -0,0 +1,58 @@
+sinks:
+
+  - id: Leakages.Log.Error
+    name: Log Error
+    patterns:
+      - "(?i)(github.com/sirupsen/logrus).*[.](error)(f?)"
```

Comment:
Do we get methodFullName for these github libaries like above?

Reply:
Here is one sample methodFullName: `github.com/sirupsen/logrus.FieldLogger.Errorf`

Reply:
For code snippet you can follow: https://golangdocs.com/logging-in-go-logrus-package